modules/telco-core-rds-container.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/core/telco-core-ref-crs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-core-rds-container_{context}"]
+= Extracting the telco core reference design configuration CRs
+
+You can extract the complete set of custom resources (CRs) for the telco core profile from the `telco-core-rds-rhel9` container image. 
+The container image has both the required CRs, and the optional CRs, for the telco core profile.
+
+.Prerequisites
+
+* You have installed `podman`.
+
+.Procedure
+
+* Extract the content from the `telco-core-rds-rhel9` container image by running the following commands:
++
+[source,terminal]
+----
+$ mkdir -p ./out
+----
++
+[source,terminal]
+----
+$ podman run -it registry.redhat.io/openshift4/telco-core-rds-rhel9:v4.16 | base64 -d | tar xv -C out
```

Comment:
Should the v4.16 be hardcoded here? Not sure what the alternative would be.

Reply:
Good point, I think it should be in this instance though.